src/PolyhedralGeometry/iterators.jl

```diff
@@ -224,9 +378,9 @@ function matrix_for_polymake(iter::SubObjectIterator; homogenized=false)
 end
 
 # primitive generators only for ray based iterators
-matrix(R::FlintIntegerRing, iter::SubObjectIterator{RayVector{Polymake.Rational}}) =
+matrix(R::FlintIntegerRing, iter::SubObjectIterator{<:RayVector}) =
```

Comment:
```suggestion
matrix(R::FlintIntegerRing, iter::SubObjectIterator{RayVector{fmpq}}) =
```
I don't think we want to call primitive for non-rational rays.